src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -2131,10 +2131,10 @@ exprt c_typecheck_baset::do_special_functions(
     if(expr.arguments().size() == 2)
     {
       if(
-        expr.arguments()[1].type() != ID_unsignedbv &&
-        expr.arguments()[1].type() != ID_signedbv)
+        expr.arguments()[1].type().id() != ID_unsignedbv &&
+        expr.arguments()[1].type().id() != ID_signedbv)
       {
-        err_location(f_op);
+        error().source_location = f_op.source_location();
```

Comment:
I'd prefer for those changes to be squashed into the earlier commit as otherwise that code wouldn't compile (which makes bisecting hard).